src/Network/Broadcast/OutboundQueue.hs

```diff
@@ -25,6 +25,7 @@
 {-# LANGUAGE ScopedTypeVariables       #-}
 {-# LANGUAGE StandaloneDeriving        #-}
 {-# LANGUAGE TupleSections             #-}
+{-# OPTIONS_GHC -fno-warn-redundant-constraints #-}
```

Comment:
Do we really need this? We'd prefer to keep it clean and fix the redundant constraints.

Reply:
Ah! This was needed because the minute I swapped a `Lens` for a `Getter` GHC complained about the unused `Contravariant` constraint, which was probably pulled in scope by the `Getter` definition. No clue how to fix that, I'd confess 😁